cv_dv_utils/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
```

Comment:
I doubt very much that we need this.  To date we have not created any stand-along releases of CORE-V-VERIF, except those that are related to a specific release of a core.

Reply:
How do we keep track of changes within the cv_dv_utils modules?

Reply:
Git automatically maintains commit logs and thus far this has been sufficient.  If in the future we support specific releases of CORE-V-VERIF, we would probably follow the strategy deployed by the cores.

Reply:
unless it is a strong rule, I prefer to keep the CHANGELOG.md. Let me know?

Reply:
No it is not a strong rule.  Up to this point we have never needed it, and files such as `changelogs` tend to suffer from a lack of attention over time.  So I would prefer not to have it, but won't demand we delete it.